configure.ac

```diff
@@ -3091,7 +3091,6 @@ AC_CONFIG_FILES([
 	tools/apparmor/Makefile
 	tools/helper-mux/Makefile
 	tools/purge/Makefile
-	tools/squidclient/Makefile
```

Comment:
```
doc/manuals/manuals.pot:msgid "squidclient - A simple HTTP web client tool"
...
doc/manuals/ar.po:msgid "squidclient - A simple HTTP web client tool"
```

What about translations? Does this PR need to strip squidclient references like the ones shown above? Or will that happen automagically somehow? CC: @yadij.

Reply:
automagic

Reply:
Great, resolving.